tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27,py33,py34,py35,py36
+
+[testenv]
+deps=pytest
```

Comment:
Right I did this on another package I'm fixing right now. Wasn't aware it was an option till I was looking through the docs more. I can update here I guess, ultimately there's so few test requirements I don't think it matters one way or another.